app/assets/javascripts/preferences/panes/account/LogOutView.tsx

```diff
@@ -0,0 +1,161 @@
+import { Button } from '@/components/Button';
+import { ConfirmationDialog } from '@/components/ConfirmationDialog';
+import {
+  PreferencesGroup,
+  PreferencesSegment,
+  Subtitle,
+  Text,
+  Title,
+} from '@/preferences/components';
+import { WebApplication } from '@/ui_models/application';
+import { FunctionComponent } from 'preact';
+import { useState } from 'preact/hooks';
+
+const OtherSessionsLogoutDialog: FunctionComponent<{
+  logout: () => Promise<void>;
+  closeDialog: () => void;
+}> = ({ logout, closeDialog }) => {
+  const [errorMessage, setErrorMessage] = useState<string | undefined>(
+    undefined
+  );
+
+  return (
+    <ConfirmationDialog title={<Title>End other sessions?</Title>}>
+      <Text>
+        The associated app will be signed out and all data removed from the
+        device when it is next launched. You can sign back in on that device at
+        any time.
+      </Text>
+      <div className="min-h-6" />
+      <div className="flex flex-row items-center w-full">
+        <Button
+          type="normal"
+          label="Cancel"
+          onClick={() => {
+            closeDialog();
+          }}
```

Comment:
```suggestion
          onClick={closeDialog}
```

Reply:
This has been changed to a different trigger function that accepts an ardgument.